src/index.js

```diff
@@ -1,10 +1,15 @@
+import 'babel-polyfill'
```

Comment:
(Not only because of its weight, but because it pollutes the global scope which is fine for certain apps [case in point ours] but isn't something you want to apply to everyone using this library). Can it be avoided?